src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java

```diff
@@ -31,23 +32,28 @@ public void onDefault(Player player) {
         player.sendMessage(ChatColor.RED + "Please provide a valid subcommand.");
     }
 
+    @Subcommand("wand")
+    @CommandPermission("wesf.admin")
+    public void onWand(Player player) {
+        final ItemStack wand = SlimefunItem.getOptionalById("WEF_WAND").map(SlimefunItem::getItem).orElse(null);
```

Comment:
```suggestion
        final ItemStack wand = SlimefunItem.getOptionalById("WESF_WAND").map(SlimefunItem::getItem).orElse(null);
```

Reply:
Oh whoops, looks like I fixed this upstream alr